### DISCUSSION TITLE
Tuning Steps

### DISCUSSION THREAD

**Discussion by merc07** [egzumer/uvk5-chirp-driver#14] [Q&A] [answered]
Is there a way to allow 1.25 step ? reason being I am unable to add channel such as 27.60125 for recieve
It is available in your 0.21 firmware but doesn't show in the selection in Chirp v0.6
There are others also, 0.5, 0.25, 0.05....
Thanks !

**Comment by egzumer** [comment]
![Screen Shot 12-30-23 at 02 01 PM](https://github.com/egzumer/uvk5-chirp-driver/assets/14902414/cf230f00-0cb7-47d1-9b51-025a001ef822)

**Comment by merc07** [comment]
I must be missing something here..
![image](https://github.com/egzumer/uvk5-chirp-driver/assets/33531315/07587a70-38e8-4935-b209-2047c6984fc4)
![image](https://github.com/egzumer/uvk5-chirp-driver/assets/33531315/c52a210e-db57-4c9e-8200-3faaaf4d3ae3)

**Reply by egzumer** [reply]
Choose the egzumer driver and not the plain UV-K5

**Comment by merc07** [comment]
Thanks, that worked - I was just using 'Currently you can use modified version of CHIRP from [releases](https://github.com/egzumer/uvk5-chirp-driver/releases).'

Thought that had the updates

Thanks for your speedy responce !

**Reply by egzumer** [reply, marked as answer]
It has the updates, but those 2 are separate drivers
